src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/Serialization/CollectionTests.cs

```diff
@@ -26,77 +26,6 @@ protected CollectionTests_Metadata(JsonSerializerWrapperForString serializerWrap
         {
         }
 
-        [Fact]
-        public override async Task ReadSimpleKeyValuePairPartialData()
```

Comment:
Why are these tests deleted? My understanding is that behavior should not change in these cases.

Reply:
Since the tests are the same for the serialization and the source gen there is not a need to override them with the same implementation.